packages/config/src/config/processing.job.ts

```diff
@@ -1,15 +1,20 @@
+import { TileMatrixSet } from '@basemaps/geo';
 import { BaseConfig } from './base.js';
 
-export enum JobStatus {
-  Processing = 'processing',
-  Complete = 'complete',
-}
-
-export type ConfigProcessingJob = ProcessingJob | ProcessingJobFailed;
+export type ConfigProcessingJob = ProcessingJob | ProcessingJobComplete | ProcessingJobFailed;
 
 export interface ProcessingJob extends BaseConfig {
-  /** Job Status for the imagery importing batch jobs */
-  status: JobStatus;
+  status: 'processing';
+}
+
+export interface ProcessingJobComplete extends BaseConfig {
+  status: 'complete';
+  /** Processed Imagery projection */
+  tileMatrix: TileMatrixSet;
```

Comment:
you cannot store the full class inside the database? this should be a `string`

Reply:
Do you mean the tileMatrix.identifier? or just String the whole object?

Reply:
> Do you mean the tileMatrix.identifier? or just String the whole object?

Yup just the identifier, we could type this as `NZTM2000Quad | WebMercatorQuad` as those are the only two we should be processing!